scripts/mujoco/ant/create_dataset.py

```diff
@@ -25,40 +24,27 @@ def __call__(self, env, **kwargs):
 
         return step_data
 
-
-DATASET_NAME = "ant-v5-expert-tuned-v0"
-dataset = None
-
-# Check if dataset already exist
+DATASET_NAME = "mujoco/ant/expert-v0"
 assert DATASET_NAME not in minari.list_local_datasets()
 
-# Create Environment
-env = gym.make("Ant-v5", include_cfrc_ext_in_observation=False, max_episode_steps=1000)
-# add callback to add cfrc_ext to the obs space
-collector_env = DataCollector(
-    env,
+env = gym.make("Ant-v5", max_episode_steps=1000)
```

Comment:
, include_cfrc_ext_in_observation=False,

Reply:
Ah, this may be a problem then, because when you will call `.recover_environment()` on the dataset, you will get the environment with the flag set to False.
Two solutions:
1) Remove again the StepDataCallback and drop the  extra observations with a Wrapper after DataCollector.
2) Retrain the model with the extra observation. We don't care about having the best policy, just a policy performing good is okay. We can also use the default train of RLZoo (which I believe is done for the policy we have on HF)

Reply:
Rlzoo is trained for mujoco-v3 enviroments

Reply:
Okay, can you merge this PR and do it, as I cannot test it?
Thank you